source/reference/compatibility.txt

```diff
@@ -248,7 +249,22 @@ and will be removed in a next version.
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - 9.0+
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     -
+     -
+     -
+     -
+
    * - 8.0 thru 9.0
```

Comment:
S: while you're in here, change this to be the full word (applies where relevant)
```suggestion
   * - 8.0 through 9.0
```